sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/download/DownloadSamplesInput.groovy

```diff
@@ -0,0 +1,22 @@
+package life.qbic.business.samples.download
+
+/**
+ * <b>Input interface for the {@link DownloadSamples} feature</b>
+ *
+ * <p>Provides methods to trigger the fetching of sample codes with available data for one project</p>>
+ *
+ * @since 1.0.0
+ */
+interface DownloadSamplesInput {
+
+    /**
+     * This method calls the output interface with the codes of samples in the project
+     * that have available data attached
+     * In case of failure the output interface failure method is called.
+     *
+     * @param projectCode a code specifying the samples that should be considered
+     * @since 1.0.0
+     */
+    void fetchSampleCodesWithData(String projectCode)
```

Comment:
What do you think about renaming this method to
`requestSampleCodes`? The context gives most information and we do not need the `fetch` detail here.